code/model/modelanimation.cpp

```diff
@@ -35,13 +35,17 @@ namespace animation {
 		
 		if (applyOnly) {
 			for (const auto& animation : m_submodelAnimation) {
-				ModelAnimationData<> base = animation->s_initialData.at({ pmi->id, animation->m_submodel });
-				ModelAnimationData<true>& previousDelta = (*applyBuffer)[animation->m_submodel].second;
-				(*applyBuffer)[animation->m_submodel].first = animation.get();
+				std::pair<int,int> keypair = {pmi->id, animation->m_submodel};
 
-				base.applyDelta(previousDelta);
-				ModelAnimationData<true> delta = animation->play(instanceData.time, instanceData.time, ModelAnimationDirection::FWD, pmi, base, true);
-				previousDelta.applyDelta(delta);
+				if(animation->s_initialData.count(keypair)>0){
```

Comment:
Pedantic, I know, but please put spaces around the `>` here and below

Reply:
Done